packages/block-editor/src/components/dimension-control/index.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { isFunction } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	Icon,
+	SelectControl,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+import {
+	Fragment,
+} from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import sizesTable, { findSizeBySlug } from './sizes';
+
+export function DimensionControl( props ) {
+	const { label, icon, sizes = sizesTable, currentSize, onSpacingChange, onReset, className = '' } = props;
```

Comment:
I guess this may be used on other things besides spacing, should we call the prop onSpacingChange simply onChange?

Reply:
Fixed